recipes/pyclipper/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pyclipper" %}
+{% set version = "1.1.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 8a8b6018d53fcce291f78dedca19994f82695eed3a2c9eff275691d4ed9aab51
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - setuptools_scm_git_archive
+    - setuptools_scm
+
+  run:
+    - python
+    - setuptools
+
+test:
+  commands:
+    - python -c "import pkg_resources; pkg_resources.require('pyclipper')"
+  imports:
+    - pyclipper
+
+about:
+  home: https://github.com/fonttools/pyclipper
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
This is not enough. Since this python library is vendoring clipper C++ library, you need to include their license too. You need to add a file with the copyright notice, https://github.com/fonttools/pyclipper/blob/master/pyclipper/clipper.hpp#L7 and the boost license text.

Reply:
Sorry but its not immediately obvious to include multiple licences. Are there any examples you can give me?

Reply:
See https://github.com/conda/conda-build/pull/3600/files#diff-3cd4e1d5ae8274c9e3f742ab550d60d6

Reply:
Is this sufficient?